### PR TITLE
Proofreading callback docstrings through `checkpoint_saver.py`

### PR DESCRIPTION
**Pull request by mcneela**

### CODE REVIEW

File: composer/callbacks/callback_hparams.py [hunk 1]
```diff
@@ -128,20 +129,21 @@ class MLPerfCallbackHparams(CallbackHparams):
...
-        division (str, optional): Division of submission. Currently only ``open`` division supported.
-        metric_name (str, optional): name of the metric to compare against the target. Default: ``Accuracy``.
+        division (str, optional): Division of submission. Currently only ``open`` division supported. Default: ``"open"``.
+        metric_name (str, optional): name of the metric to compare against the target. Default: ``"Accuracy"``.
         metric_label (str, optional): label name. The metric will be accessed via ``state.current_metrics[metric_label][metric_name]``.
-        submitter (str, optional): Submitting organization. Default: MosaicML.
+            Default: ``"eval"``.
+        submitter (str, optional): Submitting organization. Default: ``"MosaicML"``.
         system_name (str, optional): Name of the system (e.g. 8xA100_composer). If
             not provided, system name will default to ``[world_size]x[device_name]_composer``,
```
Comment (hanlint) [inline composer/callbacks/callback_hparams.py:141]: not provided -> `None`

**Review by hanlint — APPROVED** [review]
LGTM, with a few changes

File: composer/callbacks/callback_hparams.py [hunk 2]
```diff
@@ -181,22 +183,27 @@ class CheckpointSaverHparams(CallbackHparams):
 
     Args:
         save_folder (str, optional): See :class:`~.CheckpointSaver`.
+            Default: ``"{run_name}/checkpoints"``.
```
[the remark this answers is not in the document]
Comment (mcneela) [reply composer/callbacks/callback_hparams.py]: Fixed!

File: composer/callbacks/callback_hparams.py (hunk 1 above)
Comment (hanlint) [reply composer/callbacks/callback_hparams.py]: oops yes i mean line 141

**Pull request merged**